src/Symfony/Component/Messenger/Bridge/Beanstalkd/README.md

```diff
@@ -0,0 +1,12 @@
+Beanstalkd Messenger
+===============
+
+Provides Beanstalkd integration for Symfony Messenger.
+
```

Comment:
Can you add the DSN pattern?

Reply:
I've added the DSN "pattern". There aren't any in the other bridges in the README so hopefully it's ok how I've written the pattern.